docs/content/preview/develop/quality-of-service/transaction-priority.md

```diff
@@ -157,3 +162,85 @@ yugabyte=> select * from account;
   </tr>
 
 </table>
+
+### Show transaction priority types
+
+The following is another example which demonstrates the usage of `yb_transaction_priority` based on the priority types.
+
+1. From an active [ysqlsh](../../../admin/ysqlsh/#starting-ysqlsh) shell, create a table as follows:
+
+   ```sql
+   CREATE TABLE test_scan (i int, j int);
+   ```
+
+1. Start by setting the lower and upper bound values for your transaction.
+
+   ```sql
+   set yb_transaction_priority_lower_bound = 0.4;
+   set yb_transaction_priority_upper_bound = 0.6;
+   ```
+
+1. In a transaction block, perform an insert and view the transaction priority as follows:
+
+   ```sql
+   BEGIN TRANSACTION;
+   INSERT INTO test_scan (i, j) values (1, 1), (2, 2), (3, 3);
+   show yb_transaction_priority;
+   COMMIT;
+   ```
+
+   ```output
+             yb_transaction_priority
+   -------------------------------------------
+    0.620638447 (Normal priority transaction)
+   (1 row)
+   ```
+
+1. In the next transaction block, perform a `SELECT` which results in a high priority transaction.
+
+   ```sql
+   set yb_transaction_priority_lower_bound = 0.4;
+   set yb_transaction_priority_upper_bound = 0.4;
+   BEGIN TRANSACTION;
+   SELECT i, j FROM test_scan WHERE i = 1 FOR UPDATE;
+   show yb_transaction_priority;
+   COMMIT;
+   ```
+
+   ```output
+       yb_transaction_priority
+   -------------------------------------------
+    0.400000000 (High priority transaction)
```

Comment:
As followed from above, this value will be randomly chosen between between the lower and upper bound.